### DISCUSSION TITLE
My first game demo

### DISCUSSION THREAD

**Discussion by rudiHammad** [axmolengine/axmol#1263] [Show and tell]
Hello,

I did this in cocos2dx so not technically axmol, but since it is pretty much the same thing I though to post it here too.

https://www.youtube.com/watch?v=_nJy1ytX7CA
https://vimeo.com/842251252?share=copy

It was more like a test drive to get familiar with the framework and see what challenges are face when starting in game development.
Hope you like it. 
Cheers
 
edit: link fixed (thanks aismann)

**Comment by aismann** [comment]
> Hello,
> 
> I did this in cocos2dx so not technically axmol, but since it is pretty much the same thing I though to post it here too.
> 
> [https://vimeo.com/842251252?share=copy](url)
> 
> It was more like a test drive to get familiar with the framework and see what challenges are face when starting in game development. Hope you like it. Cheers
> 
> ps: not sure why, but if i click on the link it gives a 404 error, but if you copy the link in the web works

Your link is wrong (Copy the link (text) ist the "raw" text)
![image](https://github.com/axmolengine/axmol/assets/8652787/7d86d467-9d85-416a-bd42-2252d27977a1)

**Comment by aismann** [comment]
@rudiHammad 
Sorry I dont want to create a login on: 'vimeo.com'  can you use 'youtube'? 
![image](https://github.com/axmolengine/axmol/assets/8652787/9722b006-03ce-4c3b-a993-6d3d89bb5625)

**Reply by rudiHammad** [reply]
sure, I am uploading it now. Upload will finish in 40 min. I'll add the link in the first post when ready

**Comment by 10100010010001001010** [comment]
I notice fps drops, is it really so?

**Comment by rudiHammad** [comment]
yes, I was learning while making the demo. This was more a training exercice than a game, and there are things that are not done properly.
The frame drops happen when creating to many sprites at the same time off screen. I should have maybe creating a load screen to load the heavy objects, or maybe reuse the sprites that go off screen instead of deleting and creating again.
Since it is not meant to be released I took note of things like that to improve on the next project.
